README.md

```diff
@@ -1,37 +1,90 @@
-# VLCKit
+<img src="./logo.svg" alt="VLCKit logo" height="70" >
```

Comment:
Needs to be uploaded to `videolan.org/images`

Reply:
do you have a preview for this ?

Reply:
never mind just saw it in your repo :D

Reply:
Here ya go :D https://github.com/mkchoi212/vlckit/blob/7172ae66c2d333d9adb4b7745fc4f73455954c91/README.md

> btw  for future references, you can go to `Files changed -> View` to see file previews